docker-compose.yml

```diff
@@ -13,11 +13,11 @@ services:
     volumes_from:
       - appdata:ro
     depends_on:
-      - php
+      - magento2-php
 
 
-  php:
-    build: config/php/image
+  magento2-php:
```

Comment:
It's not good if you change the container name. If so, you also have to change all references e.g. https://github.com/arvatoSCM/dockerize-magento2/blob/develop/config/nginx/sites-enabled/default.conf#L2